pxr/usd/sdf/namespaceEdit.h

```diff
@@ -146,6 +143,7 @@ struct SdfNamespaceEditDetail :
                            const std::string& reason);
 
     SDF_API bool operator==(const SdfNamespaceEditDetail& rhs) const;
```

Comment:
Just a nitpick. Since == uses "rhs", != should use "rhs" too

Reply:
Done